deeppavlov/vocabs/wiki_sqlite.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, Any, Optional
+import sqlite3
+
+from deeppavlov.core.models.component import Component
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.data.utils import download
+
+DB_URL = 'http://lnsigo.mipt.ru/export/datasets/wikipedia/wiki_full.db'
+
+
+@register('wiki_sqlite_vocab')
+class WikiSQLiteVocab(Component):
+    """
+    Get SQlite documents by ids.
+    """
+
+    def __init__(self, data_dir: str = '', data_url: str = DB_URL, **kwargs):
+        """
+        :param data_dir: a directory name where DB is located
+        :param data_url: an URL to SQLite DB
+        """
+        download_dir = expand_path(data_dir)
+        download_path = download_dir.joinpath(data_url.split("/")[-1])
+        download(download_path, data_url, file_exists=False)
```

Comment:
Can we rename `file_exists` to something like `force_download` or `redownload`?

Reply:
Done